tests/test_wave_03.py

```diff
@@ -59,7 +59,7 @@ def test_friends_unique_movies_not_duplicated():
     # ****** Add assertions here to test that the correct movies are in friends_unique_movies **********
     # **************************************************************************************************
 
-@pytest.mark.skip()
+
```

Comment:
Missing assertions here.  This test should check that the list contains the correct 3 movies.